src/DynamoCoreWpf/UI/Prompts/FunctionNamePrompt.xaml.cs

```diff
@@ -49,6 +57,18 @@ void OK_Click(object sender, RoutedEventArgs e)
           }
       }
 
+      // Check if the file name contains any special non-printable chatacters.
+      bool IsFileNameValid(string fileName)
+      {
+          // Some other extra characters that are to be checked. 
+          char[] invalidCharactersFileName = { '#', '%', '&', '.', ' ' };
```

Comment:
Added these extra characters that were not being checked by the Path.GetInvalidFileNameChars() function. 
Made sure these special characters are not allowed: https://docs.microsoft.com/en-us/previous-versions/6f69bkd1(v=vs.140)

Reply:
Yes, moved the function to PathHelper.cs.